### PR TITLE
Fix release kind detection

### PR DESCRIPTION
**Pull request by ivaradi**
The changes last made to the Ubuntu/Debian build scripts do not work well on the stable branches, as it is not detected properly if we are dealing with a release or not. This patch fixes this problem.

### CODE REVIEW

**Review by claucambra — APPROVED** [review]

**Comment by ivaradi** [comment]
Hi @claucambra , thanks for the approval. However, as usual some of the checks are failing, and I cannot merge it. What is more annoying is that the Drone CI does not run, but this time it would be actually useful to see if my changes work in that environment too. Is there something you could do about it?

**Comment by claucambra** [comment]
> Hi @claucambra , thanks for the approval. However, as usual some of the checks are failing, and I cannot merge it. What is more annoying is that the Drone CI does not run, but this time it would be actually useful to see if my changes work in that environment too. Is there something you could do about it?

Looks like there is a hold-up with drone

Could you sign-off the commit? This would fix the DCO check, thanks!

**Comment by mgallien** [comment]
> Hi @claucambra , thanks for the approval. However, as usual some of the checks are failing, and I cannot merge it. What is more annoying is that the Drone CI does not run, but this time it would be actually useful to see if my changes work in that environment too. Is there something you could do about it?

@ivaradi 
for security reason, we cannot let drone run for branches from outside repositories (i.e. clones)
I think you have write access to this public repository, you can use this to let drone run over your PR

**Comment by ivaradi** [comment]
> @ivaradi for security reason, we cannot let drone run for branches from outside repositories (i.e. clones) I think you have write access to this public repository, you can push your branch in it to let drone run over your PR

OK, I have created a new pull request from a branch in this repository: #7031

**Comment by ivaradi** [comment]
> @ivaradi
> for security reason, we cannot let drone run for branches from outside repositories (i.e. clones)
> I think you have write access to this public repository, you can push your branch in it to let drone run over your PR

OK, I have created a new PR from a branch in this repository: #7031 

Closing this one.

**Pull request closed without merging**
